File: PHP/nilaiAkhirdanStatus.php
<?php
$tugas = readline("Masukan nilai tugas: ");
$uts = readline("masukan nilai UTS: ");
$uas = readline("masukan nilai UAS: ");

$nilaiAkhir = (0.3 * $tugas) + (0.3 * $uts) + (0.4 * $uas);

echo "Nilai AKhir = $nilaiAkhir\n";
if ($nilaiAkhir >= 60)
    echo "Status = Lulus\n";
else
    echo "Status = tidak lulus\n";
?>

File: C/C/nilai.akhir.dan.status.cpp
#include<iostream>
using namespace std;

int main(){
    float tugas, uts, uas, akhir;
    cout << "masukan nilai tugas: ";
    cin >> tugas;
    cout << "masukan nilai uts: ";
    cin >> uts;
    cout >> "masukan nilai uas: ";
    cin >> uas;

    nilaiAkhir = (0.3 * tugas) + (0.3* uts) + (0.4 * uas);

    cout<< "nilai akhir=" <<nilaiAkhir<<endl;
    if (nilaiAkhir>=60)
    cout<<"Status:  Lulus";
    else
    cout<<"Status: tidak lulus";
}